apps/cli/src/commands/run.ts

```diff
@@ -54,6 +54,25 @@ export default class Run extends BaseCommand<typeof Run> {
             description: "port to listen for incoming connections",
             default: 8080,
         }),
+        cpus: Flags.integer({
+            description:
+                "Define the number of CPUs to use (eg.: 1) for the rollups-node, 0 (zero) means unlimited",
+            summary: "number of cpu limits for the rollups-node",
+            default: 4,
```

Comment:
I think the option should be optional, without any default value.
It will default to whatever docker compose defines as a non-defined resources: limitation.

Reply:
fixup sent 9c565b3

Reply:
No, the default is not `0`. It should be `undefined`.

Reply:
The way I exposed to the cartesi cli, if you define `0` it translates to undefined, since the `resources:` section will not be loaded into the final .yaml.

You can test with `cartesi run --show-config` vs. `cartesi run --show-config --cpus=4 --memory=4096`

Reply:
That is because `if(0)` is false in typescript, but it's more correct to explicitly be undefined.
`0` is different from `undefined`.

Reply:
sent fixup 4c493a1